Makefile

```diff
@@ -29,3 +29,39 @@ clippy:
 
 clean:
 	rm -f $(TEST_DIR)/*.json
+	rm -rf cairo-rs-py-env
+
+run-python-test-macos: $(COMPILED_TESTS)
+	python3 -m venv cairo-rs-py-env
+	. cairo-rs-py-env/bin/activate && \
+	pyenv install pypy3.7-7.3.9 && \
+	pyenv local pypy3.7-7.3.9 && \
+	CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib pip install fastecdsa && \
+	pip install cairo_lang==0.9.1 && \
```

Comment:
All of this belongs to a `deps` target. Also, the `&& \` is not necessary, any `make` target fails if any of its commands fails.

Reply:
Also, don't set `local`, it's more robust to pass the `PYENV_VERSION` environmental variable instead.

Reply:
`venv` activation does belong here tho. Deactivation requires a little bit more care for failing tests.

Reply:
Moved env setup to deps
The reason for using `&& \ `is that if not used then the virtual environment is not activated when `maturin develop` is called, causing failure.